apps/ticket/src/app/(pages)/event/[eventId]/booking/[bookingId]/confirmed/_clientBoundary/EventInfoClient/index.tsx

```
@@ -0,0 +1,29 @@
+"use client";
+
+import classNames from "classnames/bind";
+
+import { Typography } from "@permit/design-system";
+
+import styles from "./index.module.scss";
+
+const cx = classNames.bind(styles);
+
+type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)